src/vanilla/internals.ts

```diff
@@ -996,7 +1010,7 @@ export {
   // Export internal functions
   //
   buildStore as INTERNAL_buildStoreRev2,
-  getBuildingBlocks as INTERNAL_getBuildingBlocksRev2,
+  getExternalBuildingBlocks as INTERNAL_getBuildingBlocksRev2,
```

Comment:
Let's simply call the new one `getBuildingBlocks` as the user of this function doesn't care if it's internal or enhanced. It's also consistent for readability.
```suggestion
  getBuildingBlocks as INTERNAL_getBuildingBlocksRev2,
```